src/file_resolver.rs

```diff
@@ -236,7 +236,7 @@ impl FileResolver {
             return Ok(ResolvedFile {
                 full_path,
                 base,
-                import_no: 0,
+                import_no: Some(0),
```

Comment:
copy of import_no in cache

Reply:
Where can I get this? I don't think this is accessible through FileResolver (it's in File which is stored in Namespace).